dbt/include/databricks/macros/materializations/incremental/validate.sql

```diff
@@ -35,7 +35,7 @@
     Use the 'merge' or 'replace_where' strategy instead
   {%- endset %}
 
-  {% if raw_strategy not in ['append', 'merge', 'insert_overwrite', 'replace_where', 'microbatch'] %}
+  {% if raw_strategy not in adapter.valid_strategies %}
```

Comment:
I saw above method is `valid_incremental_strategies`?

Reply:
good call out, probably about to fail the E2E tests lol.